addons/l10n_ch/tests/test_gen_isr_reference.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common, Form
+
+QR_IBAN = 'CH21 3080 8001 2345 6782 7'
+ISR_SUBS_NUMBER = "01-162-8"
+
+
+class TestGenISRReference(common.SavepointCase):
```

Comment:
You should probably rather inherit this instead https://github.com/odoo/odoo/blob/13.0/addons/account/tests/account_test_savepoint.py#L13